### PR TITLE
Convert win.proc.cpu* to 0-100% range instead of being per core.

### PR DESCRIPTION
**Pull request by gbrayut**

### CODE REVIEW

File: collectors/processes_windows.go [hunk 1]
```diff
@@ -40,6 +40,17 @@ func c_windows_processes() (opentsdb.MultiDataPoint, error) {
 		iis_dst = nil
 	}
 
+	var numberOfLogicalProcessors uint64
+	var core_dst []Win32_Processor
+	var core_q = wmi.CreateQuery(&core_dst, `WHERE Name <> '_Total'`)
```
Comment (madelynnblue) [inline collectors/processes_windows.go]: Do we need the WHERE clause here? My tests show we don't, so this could be an empty string.

File: collectors/processes_windows.go [hunk 2]
```diff
@@ -40,6 +40,17 @@ func c_windows_processes() (opentsdb.MultiDataPoint, error) {
...
+	err = queryWmi(core_q, &core_dst)
+	if err != nil {
+		return nil, err
+	}
+	for _, y := range core_dst {
+		numberOfLogicalProcessors = uint64(y.NumberOfLogicalProcessors)
```
Comment (madelynnblue) [inline collectors/processes_windows.go:56]: How does this work with machines with two physical processors (like ny-rdp01)? The total number of logical processors there is 32, not 16 (as this would compute).

File: collectors/processes_windows.go [hunk 3]
```diff
@@ -90,9 +101,9 @@ func c_windows_processes() (opentsdb.MultiDataPoint, error) {
 
 		//Use timestamp from WMI to fix issues with CPU metrics
 		ts := TSys100NStoEpoch(v.Timestamp_Sys100NS)
-		AddTS(&md, "win.proc.cpu", ts, v.PercentPrivilegedTime/NS100_Seconds, opentsdb.TagSet{"name": name, "id": id, "type": "privileged"}, metadata.Counter, metadata.Pct, descWinProcCpu_priv)
-		AddTS(&md, "win.proc.cpu", ts, v.PercentUserTime/NS100_Seconds, opentsdb.TagSet{"name": name, "id": id, "type": "user"}, metadata.Counter, metadata.Pct, descWinProcCpu_user)
-		AddTS(&md, "win.proc.cpu_total", ts, v.PercentProcessorTime/NS100_Seconds, opentsdb.TagSet{"name": name, "id": id}, metadata.Counter, metadata.Pct, descWinProcCpu_total)
+		AddTS(&md, "win.proc.cpu", ts, v.PercentPrivilegedTime/NS100_Seconds/numberOfLogicalProcessors, opentsdb.TagSet{"name": name, "id": id, "type": "privileged"}, metadata.Counter, metadata.Pct, descWinProcCpu_priv)
```
Comment (madelynnblue) [inline collectors/processes_windows.go:112]: What if, for some reason, no results are returned from the WMI query above, and thus we end up with 0? The divide-by-zero should be avoided with a check above.
